experiment/simulation/js/fa.js

```diff
@@ -0,0 +1,333 @@
+import { registerGate } from "./main.js";

```

Comment:
Please use strict mode for all JavaScript.

Reply:
done